### PR TITLE
add support for bundling

### PR DESCRIPTION
**Pull request by atulsriv**

### CODE REVIEW

File: bxsolana/provider/ws.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ class WsProvider(Provider):
     # noinspection PyMissingConstructor
     def __init__(
         self,
-        endpoint: str = constants.MAINNET_API_NY_WS,
+        endpoint: str = constants.TESTNET_API_WS,
```
Comment (DimaCantemir) [inline bxsolana/provider/ws.py]: revert to prod please

File: bxsolana/transaction/private_txs.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+from solders import transaction as solders_tx
+from solders.hash import Hash
+from solders.keypair import Keypair
+from solders.message import Message
+from solders.pubkey import Pubkey
+from solders.system_program import transfer, TransferParams
+from solders.transaction import Transaction
+
+BloxrouteTipWallet = pk.Pubkey.from_string(
+    "AFT8VayE7qr8MoQsW3wHsDS83HhEvhGWdbNSHRKeUDfQ"
```
Comment (DimaCantemir) [inline bxsolana/transaction/private_txs.py]: I would not hardcode but document and let the user get the fresh one from our docs

File: example/bundles/main.py [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+market_addr = "9wFFyRfZBsuAha4YcuxcXLKwMxJR43S7fPfQLusDBzvT"  # SOL/USDC
+order_side = proto.Side.S_ASK
+order_type = OrderType.OT_LIMIT
+order_price = 170200
+order_amount = 0.1
+
+
+async def main():
+    # await ws()
```
Comment (DimaCantemir) [inline example/bundles/main.py]: uncomment or remove please

File: example/bundles/main.py [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+    await grpc()
+    # await http()
```
Comment (DimaCantemir) [inline example/bundles/main.py]: same as above

**Review by DimaCantemir — CHANGES_REQUESTED** [review]

**Review by tinystarinagalaxy — APPROVED** [review]

**Review by etienneblxr — APPROVED** [review]

**Review by DimaCantemir — APPROVED** [review]

**Pull request merged**